src/dto/User/create.user.dto.ts

```diff
@@ -0,0 +1,25 @@
+import Joi from 'joi';
+
+const passwordRegExp = () => new RegExp('^[a-zA-Z0-9]{3,30}$');
```

Comment:
To nie musi być funkcja, może być po prostu const
```suggestion
const passwordRegExp = new RegExp('^[a-zA-Z0-9]{3,30}$');
```